students/ZhuojingXie/lesson09/mailroom5.py

```diff
@@ -0,0 +1,111 @@
+import sys
+import os
+from collections import defaultdict
+donor_data = defaultdict(list,{'Andy':[960,256,123.5,40],'Bryce':[30,45,27],
+                            'Charile': [25,50], 'David':[10], 'Elaine' :[75,26]})
+
+class Donor():
+
+
+    """this is Donor class to save donor's information"""
+    def __init__(self, name, donation):
+        self.name = name
+        self.donation = donation
+
+
+    def add_donor_in_data(self):
+        return donor_data[self.name].append(self.donation)
```

Comment:
you `Donor` class should provide everything about the donor like:
* name (which you have)
* list of donations (rename donation to donations, your var names should reflect plural vs singular)
* average, total, number of donations (can be `@property`)
* ability to add donation